Microsoft.Toolkit.Uwp.UI.Controls/Carousel/Carousel.cs

```diff
@@ -234,6 +234,11 @@ private void FocusContainerFromIndex(int index)
         }
 
         /// <summary>
+        /// Occurs when the selected item has changed
+        /// </summary>
+        public event EventHandler<object> SelectionChanged = null;
```

Comment:
Thx @skendrot for the explanation on what is AddedItems and RemovedItems :D
Eventually I'm totally disagree with you on that point. I think you are adding complexity to the control and its behavior.
But anyway, since you're the reviewers, it's up to you.